space_packet_parser/xtce/definitions.py

```diff
@@ -493,64 +532,30 @@ def packet_generator(
         """
         root_container_name = root_container_name or self.root_container_name
 
-        # Used to keep track of any continuation packets that we encounter
-        # gathering them all up before combining them into a single packet
-        # for the XTCE to parse, lookup is by APID.
-        # _segmented_packets[APID] = [RawPacketData, ...]
-        _segmented_packets = {}
-
         # Iterate over individual packets in the binary data
-        for raw_packet_data in packets.ccsds_generator(binary_data,
-                                                       buffer_read_size_bytes=buffer_read_size_bytes,
-                                                       show_progress=show_progress,
-                                                       skip_header_bytes=skip_header_bytes):
+        for raw_packet_data in ccsds.ccsds_generator(binary_data,
```

Comment:
Should this generator be default kwarg to `packet_generator` so users can pass custom non-CCSDS generators? I know you have a vision for this but this feels like we're only halfway to being truly generic because the generic `packet_generator` is hard-coded to use the CCSDS-specific bytes generator here.